resources/assets/js/bootstrap.js

```diff
@@ -0,0 +1,130 @@
+/* global window, document, $, autosize, Messenger, moment, livestamp, swal */
+import Vue from 'vue';
+import VueResource from 'vue-resource';
+
+// App config
+const Cachet = window.Cachet || {};
+
+Cachet.Globals = {
+  host: window.location.host,
+  base_url: `${window.location.protocol}//${window.location.host}`,
+  url: document.URL,
+  env: $('meta[name="env"]').attr('content') || 'production',
```

Comment:
I reckon we inject a `script` tag with some global variables, instead of reading the meta data, this way we're not relying on the DOM to be ready?